tests/sentry/api/endpoints/test_organization_dashboard_details.py

```diff
@@ -435,7 +435,7 @@ def test_delete_permission(self):
         self.create_user_member_role()
         self.test_delete()
 
-    def test_disallow_delete_when_no_project_access(self):
+    def test_allow_delete_when_no_project_access(self):
```

Comment:
We should keep the old tests as well to protect the existing behaviour until we remove the flag and make the new permissions the default